.github/workflows/pr-check_markdownlint.yml

```
@@ -0,0 +1,37 @@
+name: Markdownlint (PR files)
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "**/*.md"
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v29.0.2
```

I feel like pinning actions to the major version is sufficient:

```suggestion
        uses: tj-actions/changed-files@v29
```

---

Agreed, you might want to do the same upstream